src/main/java/com/nha/abdm/wrapper/hrp/common/Utils.java

```diff
@@ -0,0 +1,26 @@
+package com.nha.abdm.wrapper.hrp.common;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.concurrent.ThreadLocalRandom;
+import lombok.Data;
+import org.springframework.stereotype.Component;
+
+@Component
+@Data
```

Comment:
do we need Data annotation here?

Reply:
There is no need of @Data here so, its removed